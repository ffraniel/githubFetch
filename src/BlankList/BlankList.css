.blankList {
    /* border:solid 2px black; */
    background-color: rgb(250, 250, 250);
    padding-left:4%;
    height:100vh;
    font-family: Arial, sans-serif;
}