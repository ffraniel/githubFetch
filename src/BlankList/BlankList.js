import React, { Component } from 'react';
import './BlankList.css';

class BlankList extends Component {
    render() {
        return (
            <div className="blankList">
                <p>search me</p>
            </div>
        )
    }
}

export default BlankList;