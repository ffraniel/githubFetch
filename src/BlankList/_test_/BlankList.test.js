import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import BlankList from '../BlankList';

configure({ adapter: new Adapter() });

test('BlankList renders a simple element', () => {
    const wrapper = mount(
        <BlankList />
    );
    const textP = wrapper.find('.blankList p');
    expect(textP.text()).toBe("No repos to show.");

    const rendered = renderer.create(
        <BlankList />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
})