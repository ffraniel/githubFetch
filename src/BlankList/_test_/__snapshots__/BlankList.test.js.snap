// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlankList renders a simple element 1`] = `
<div
  className="blankList"
>
  <p>
    No repos to show.
  </p>
</div>
`;
